CWE/events/portugal.txt

```diff
@@ -329,6 +767,252 @@ country_event = {
+		    limit = { culture = portuguese }
+		    consciousness = 1
+			ideology = { factor = 0.10 value = populist } 
+			ideology = { factor = 0.20 value = nationalist }
+		}
+		
+		ai_chance = { factor = 0.2 }
+	}
+
+}
+
+#Colonial Victory - now what?
+country_event = {
+  id = 8011724
+  title = "The Oversea Campaign is succesfull"
+  desc = EVT_8011724_DESC
+  picture = "endofpor"
+  fire_only_once = yes
+  
+	trigger = {
+		tag = POR
+		year = 1973
+		NOT = { year = 1990 }
+		
+		ruling_party_ideology = nationalist
+		government = populist_dictatorship
+		
+		is_vassal = no
+		war = no
+		
+		owns = 1446 # Timor
+		owns = 1498 # Macau
+		owns = 1876 # Cape Verde
+		owns = 1877 # Bissau
+		owns = 1954 # Sao Tome
+		owns = 1999 # Luanda
+		owns = 2049 # Maputo
+		
+		NOT = { exists = TIM }
+		NOT = { exists = MCA }
+		NOT = { exists = CPV }
+		NOT = { exists = GUB }
+		NOT = { exists = STP }
+		NOT = { exists = ANG }
+		NOT = { exists = MOZ }
+
+		NOT = { has_country_flag = carnation_revolution }
+	}
+  
+	mean_time_to_happen = { months = 3 }
+
+	option = {
+		name = "Establish the Portuguese Commonwealth!"
+		prestige = 50
+		
+		release_vassal = ANG
+		ANG = { 
+		    government = mandate_government
+			ruling_party_ideology = conservative
+			primary_culture = portuguese
+			add_accepted_culture = black_portuguese
+			add_accepted_culture = pardo_portuguese
+		}
+		release_vassal = MOZ
+		MOZ = { 
+		    government = mandate_government
+			ruling_party_ideology = conservative
+			primary_culture = portuguese
+			add_accepted_culture = black_portuguese
+			add_accepted_culture = pardo_portuguese
+		}
+		release_vassal = GUB
+		GUB = { 
+		    government = mandate_government
+			ruling_party_ideology = conservative
+			primary_culture = portuguese
+			add_accepted_culture = black_portuguese
+			add_accepted_culture = pardo_portuguese
+		}
+		release_vassal = STP
+		STP = { 
+		    government = mandate_government
+			ruling_party_ideology = conservative
+			primary_culture = portuguese
+			add_accepted_culture = black_portuguese
+			add_accepted_culture = pardo_portuguese
+		}
+		release_vassal = CPV
+		CPV = { 
+		    government = mandate_government
+			ruling_party_ideology = conservative
+			primary_culture = portuguese
+			add_accepted_culture = black_portuguese
+			add_accepted_culture = pardo_portuguese
+		}
+		release_vassal = TIM
+		TIM = { 
+		    government = mandate_government
+			ruling_party_ideology = conservative
+			primary_culture = portuguese
+			add_accepted_culture = black_portuguese
+			add_accepted_culture = pardo_portuguese
+		}
+		release_vassal = MCA
+		MCA = { 
+		    government = mandate_government
+			ruling_party_ideology = conservative
+			primary_culture = portuguese
+			add_accepted_culture = black_portuguese
+			add_accepted_culture = pardo_portuguese
+		}
+
+		ai_chance = { factor = 0.5 }
+	}
+
+	option = {
+		name = "We will maintain direct rule"
+		any_owned = { limit = { is_core = ANG } add_core = POR }
```

Comment:
Cores should not be added over colonies as I have a generic system planned to add cores over colonies by expelling natives and replacing them with primary culture so if a majority of pops are primary culture then a core is added.